themes/default/content/docs/guides/continuous-delivery/add-support-for-cicd-systems.md

```diff
@@ -63,7 +63,7 @@ The `EnvVarsToDetect` is used by the `IsCI()` in [`systems.go`](https://github.c
 
 ### Detecting Additional Metadata About A CI Build
 
-A CI build could have been triggered by a PR or a push build. In the case of the former, the linked PR number is useful so that the Pulumi Service can add a link to it in the source repo of your stack. Note that the detection is done on a best-effort basis. We support linking to the following Git-based version control systems:
+A CI build could have been triggered by a PR or a push build. In the case of the former, the linked PR number is useful so that the Pulumi Cloud can add a link to it in the source repo of your stack. Note that the detection is done on a best-effort basis. We support linking to the following Git-based version control systems:
```

Comment:
```suggestion
A CI build could have been triggered by a PR or a push build. In this case, the linked PR number is useful so that the Pulumi Cloud can link to it. Note that the detection is done on a best-effort basis. We support linking to the following Git-based version control systems:
```